torch/csrc/jit/passes/inliner.cpp

```diff
@@ -12,18 +13,34 @@ namespace prim {
 using namespace ::c10::prim;
 }
 
+GraphFunction* tryToGraphFunction(Node* n) {
+  if (n->kind() == prim::CallFunction) {
+    AT_ASSERT(n->input(0)->node()->kind() == prim::Constant);
+    auto function_constant = n->input(0)->node();
+    auto fun_type = function_constant->output()->type()->expect<FunctionType>();
```

Comment:
use expectRef() -- `const auto& fun_type = function_constant->output()->type()->expectRef<FunctionType>()`

Reply:
this is already the existing status code.. will fix in follow diff trying to land this today.